aspnetcore/razor-pages/ui-class.md

```diff
@@ -22,13 +22,13 @@ Razor views, pages, controllers, page models, [Razor components](xref:blazor/cla
 
 # [Visual Studio](#tab/visual-studio)
 
-* From the Visual Studio **File** menu, select **New** > **Project**.
-* Select **ASP.NET Core Web Application**.
-* Name the library (for example, "RazorClassLib") > **OK**. To avoid a file name collision with the generated view library, ensure the library name doesn't end in `.Views`.
-* Verify **ASP.NET Core 3.0** or later is selected.
-* Select **Razor Class Library** > **OK**.
+* From the Visual Studio **File** menu, select **New**, then select **Project**.
+* Select **Razor Class Library**, then select **Next**.
```

Comment:
Go back to the original and use `>` in lieu of  `,then select`. Do this everywhere you made that change. > is much preferred.

Reply:
Thanks. I am leaving "select" in this sentence.  It is a bit confusing having > at the end of a lot text that is not the step itself.   "* For ASP.NET 3.0, select **Support pages and views** if you need to support views. By default, only Razor Pages are supported. Select **Create**."